### DISCUSSION TITLE
Broken Functionality policy: Violation of Broken Functionality policy

### DISCUSSION THREAD

**Discussion by dexter0381** [zixpo/candybar#192] [Q&A]
Hi Everyone!

First and foremost, Thank you for the developers of Candybar. I really like the application.
However, I need your help. I hope you can help me. For the longest time. I was able to update my icon theme online on Google Playstore. However, on most recent update I had where I added more icons. It was rejected by Google with Error 

Violation: 
Broken Functionality policy: Violation of Broken Functionality policy

Details
Your app contains content that isn’t compliant with the Broken Functionality policy.
App contains icon(s) or button(s) that are not responsive or do not return content.

 I check the pre-launch reports. Here are the warnings. I really hope anyone can help me out. I dont know where to change those.

![Error1](https://github.com/user-attachments/assets/7d6a5cfc-7693-4446-8ad9-f66de07c77e7)
![Error2](https://github.com/user-attachments/assets/260c99be-02b1-45a7-b394-0192568f8f31)
![Error3](https://github.com/user-attachments/assets/197fa042-6202-497a-bf60-3aad33852c46)
![Rejected_Error](https://github.com/user-attachments/assets/47681a1e-5844-4f2c-ae86-9639b0e84d73)

**Comment by Donnnno** [comment]
Haven't seen this before, but gotta say that this looks worrying! Does Google give any more specific insight into these issues?

**Comment by dexter0381** [comment]
Yep. I just check the pre-launch reports. Here is one example. I dont know how to fix it? 
![Error_sample1](https://github.com/user-attachments/assets/97c3e1d3-0180-494c-a585-1b94ac59946f)

**Comment by Donnnno** [comment]
I'm gonna try adding the screen reader labels later :)
Looks like a good thing to add.

**Comment by dexter0381** [comment]
Thank you brother :)
Let me know anything else you need. If you need me to send the pre-launch report?

**Comment by moertel** [comment]
I know this is super late but ⚠️ warnings in the prelaunch report will not fail the review. There should be an actual red ⛔️ error somewhere that shows the specific action that went wrong.

**Comment by dexter0381** [comment]
All good now. I started again from scratch using latest version. Thank you

Sent from Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
